metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Transitive release date control for child objects of a Study
```

Comment:
This looks good please add tests for publications and web resources as well. and also patching the unreleased study to released and vice-versa.

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/f2d050aede91f1fa82253e24924f6e68a86fa038. Expanded the tests to:
- Check for Publication and WebResource
- Test patching the study from released to unreleased state

Unfortunately we can't test patching unreleased to released, because, as I mentioned during one of the previous standups, the moment the study we make study unreleased it becomes inaccessible (because it is, well, unreleased). This will be fixed when T2D-216 (Allow admins and object owners to access them before release date) is implemented.

Reply:
please see my comments in ReleaseDateAspect file

Reply:
Added tests for patching Study from unreleased to released and vice versa in https://github.com/EBIvariation/metadata-ws/pull/53/commits/8ffa5dcca48166a32ac23b29ad55df341f95f290.